packages/nimble/inst/tests/test-refactorCompilationSteps.R

```diff
@@ -0,0 +1,61 @@
+source(system.file(file.path('tests', 'test_utils.R'), package = 'nimble'))
+
+context("Testing of old vs. new generated C++ during refactoring steps")
+
+oldWarnLevel <- options('warn')
+options(warn = -1)
+
+## Known concern: ordering of asRow()/asCol() and intermediates
+
+compareOldAndNewCompilationRC <- function(input) {
```

Comment:
It would be nice to have unit tests, rather than these tests that "this unit behaves the same as it used to".